Transmission.xcodeproj/project.pbxproj

```diff
@@ -7,37 +7,37 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		0A6169A70FE5C9A200C66CE6 /* bitfield.c in Sources */ = {isa = PBXBuildFile; fileRef = 0A6169A50FE5C9A200C66CE6 /* bitfield.c */; };
+		0A6169A70FE5C9A200C66CE6 /* bitfield.c in Sources */ = {isa = PBXBuildFile; fileRef = 0A6169A50FE5C9A200C66CE6 /* bitfield.c */; settings = {COMPILER_FLAGS = "-fno-objc-arc"; }; };
```

Comment:
This is not even Obj-C code, why would you need to compile it with `-fno-objc-arc`? I presume that ARC should only be enabled for Transmission (the app) and QuickLookPlugin projects